Tests/AppcuesKitTests/Actions/AppcuesTrackActionTests.swift

```diff
@@ -48,6 +48,29 @@ class AppcuesTrackActionTests: XCTestCase {
         XCTAssertEqual(completionCount, 1)
         XCTAssertEqual(trackCount, 1)
     }
+    
+    func testExecuteWithAttributes() throws {
+        // Arrange
+        var completionCount = 0
+        var trackCount = 0
+        appcues.analyticsPublisher.onPublish = { trackingUpdate in
+            XCTAssertEqual(trackingUpdate.type, .event(name: "My Custom Event", interactive: true))
+            
+            [
+                "_builderButtonClick": true,
```

Comment:
can replace this with:

```
"boolean": true,
                                            "string": "string",
                                            "int": 10,
                                            "double": 10.5
```

to cover more lines of code if you want, did not notice it in the first pass

Reply:
Good idea!